challenge3/webapp/y-shinada/db/seeds.rb

```diff
@@ -0,0 +1,44 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+require "csv"
+
+class Seed
+  attr_reader :logger
+  attr_reader :std_logger
+
+  def initialize
+    @logger = Rails.logger
+    @std_logger = Logger.new(STDOUT)
+  end
+
+  def log_info(msg)
+    @logger.info(msg)
+    @std_logger.info(msg)
+  end
+
+  # rubocop:disable Metrics/AbcSize, Lint/UselessAssignment
+  def execute
+    Dir.glob(Rails.root.join("db", "seeds", "*")).each do |f, _i|
+      model_klass = File.basename(f, ".csv").classify.constantize
+      log_info("------------------------------")
+      log_info("truncate: #{model_klass.table_name}")
+      rs = ApplicationRecord.connection.execute "TRUNCATE TABLE #{model_klass.table_name};"
+
+      mr = []
```

Comment:
好みの問題かもしれないが
```suggestion
      mr = CSV.foreach(f, headers: true).map do |cr|
```
とする方がすっきりするかな、とも思った。